ic-os/components/upgrade/shared-resources/monitor-expand-shared-data/monitor-expand-shared-data.py

```diff
@@ -20,11 +20,10 @@
 VG_NAME = "store"
 LV_NAME = "shared-data"
 MOUNT_POINT = "/var/lib/ic/data"
-
 DEVICE_NAME = "/dev/" + VG_NAME + "/" + LV_NAME
 
-REQUIRED_AVAIL_FRACTION = 0.10
-EXPAND_FRACTION = 0.05
+# 15TiB in MiB
+MAX_LV_SIZE = 15 * 1024 * 1024
```

Comment:
The current largest data partition seems to be around 8.5TiB.
`max(node_filesystem_size_bytes{mountpoint="/var/lib/ic/data"})`